03-files/test_files.py

```diff
@@ -0,0 +1,30 @@
+import unittest
+import files
+
+class TestFiles(unittest.TestCase):
+
+    def test_read_file_contents(self):
+        self.assertEqual(files.read('test.txt'),
+"""Beautiful is better than ugly.
+Explicit is better than implicit.""")
+        self.assertEqual(len(files.read('zen.txt')), 856)
+
+    def test_file_contains(self):
+        # implement files.contains that will check whether a file contains some text
+        self.assertTrue(files.contains('better', 'test.txt'))
```

Comment:
https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertTrue Please teach people to use more precise `assert` when available. `assertIs` is better in both cases for `assertTrue` and `assertFalse`.